### PR TITLE
Update fileio.py

### PR DESCRIPTION
**Pull request by vinaysraghavan**
Updated import_outstruct to use h5py instead of hdf5storage.

#### What does this implement/fix? Explain your changes.
Should be faster for larger files and outstructs containing cell arrays

#### Any other comments?
May have trouble if the outstructs contain nested matlab data structures (cell within a cell within the outstruct field), but it should be able to handle un-nested matlab data structures and arrays

Assumes a field containing a vector of integers is an encoded string

### CODE REVIEW

**Comment by gavinmischler** [comment]
Can you add h5py to requirements.txt?

**Comment by vinaysraghavan** [comment]
I think it's already a dependency of hdf5storage, but yeah

File: naplib/io/fileio.py [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
 from tqdm import tqdm
 from hdf5storage import loadmat, savemat
+import h5py
 
 from ..data import Data
 
 ACCEPTED_CROP_BY = ['onset', 'durations']
 
-def import_outstruct(filepath, strict=True):
+def import_outstruct(filepath, strict=True, useloadmat=True, verbose=False):
```
Comment (gavinmischler) [inline naplib/io/fileio.py:12]: Can you add these new params to the docstring

**Comment by gavinmischler** [comment]
> I think it's already a dependency of hdf5storage, but yeah

oh yeah true, but still good to have it explicit in case other packages change

**Comment by gavinmischler** [comment]
Can you also add at least one test to the tests/io/test_fileio.py that uses `loadmat=False`? You can copy the format of the other import_outstruct tests

**Comment by vinaysraghavan** [comment]
> Can you also add at least one test to the tests/io/test_fileio.py that uses `loadmat=False`? You can copy the format of the other import_outstruct tests

h5py will only work on -v7.3 mat files. Are the test files in that format?

**Comment by gavinmischler** [comment]
Sorry actually rather than include two large files inside the package can you just replace the old demo_data.mat with the v7.3 version? otherwise installing the package includes an unecessary extra file

**Comment by gavinmischler** [comment]
sorry I think my comment was confusing, I meant can you rename demo_data_v73.mat to be demo_data.mat, therefore replacing the old demo_data.mat with a file that is now v7.3. Then all the tests can automatically use this file and only 1 file will be shipped with the package.

But then looks great!

**Comment by vinaysraghavan** [comment]
> sorry I think my comment was confusing, I meant can you rename demo_data_v73.mat to be demo_data.mat, therefore replacing the old demo_data.mat with a file that is now v7.3. Then all the tests can automatically use this file and only 1 file will be shipped with the package.
> 
> But then looks great!

Yup, that's exactly what I did

**Comment by gavinmischler** [comment]
> > sorry I think my comment was confusing, I meant can you rename demo_data_v73.mat to be demo_data.mat, therefore replacing the old demo_data.mat with a file that is now v7.3. Then all the tests can automatically use this file and only 1 file will be shipped with the package.
> > But then looks great!
> 
> Yup, that's exactly what I did

oh my bad, didn't see the changed filename

**Review by gavinmischler — APPROVED** [review]
Thanks for adding this!

**Comment by vinaysraghavan** [comment]
To improve test coverage, I could add a fake "chname" field with {'Fz', 'Cz', etc.} that'll cover the whole thing

**Pull request merged**